README.md

```diff
@@ -34,14 +34,25 @@ Edit this document to include your answers after each question. Make sure to lea
 
```

Comment:
Great answers to the self study questions, it shows that you have a good understanding of the things covered this week